### PR TITLE
Mobile Audio Player

### PR DESCRIPTION
**Pull request by sabeurthabti**
## UX/ Interactions

thovuxumrn

## States
screen shot 2016-12-27 at 23 51 28

screen shot 2016-12-27 at 23 54 03

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33032

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33033

File: src/components/Audioplayer/MobilePlayer.js [hunk 1]
```diff
@@ -0,0 +1,263 @@
+import React, { Component, PropTypes } from 'react';
```
Comment (sabeurthabti) [inline src/components/Audioplayer/MobilePlayer.js]: Need to make some of these functions reusable.

**Comment by sabeurthabti** [comment]
Early preview. More work still required, needs testing on real devices.

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33034

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33039

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33040

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33041

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33042

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33043

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33044

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33045

**Pull request merged**